editor/src/clj/editor/scene_visibility.clj

```diff
@@ -310,20 +311,24 @@
   (g/update-property! scene-visibility :visibility-filters-enabled? not))
 
 (defn- make-visibility-toggles-list
-  ^Region [scene-visibility]
-  (let [make-control
-        (fn [{:keys [label tag]}]
+  ^Region [app-view scene-visibility]
+  (let [keymap (g/node-value app-view :keymap)
+        command->shortcut (keymap/command->shortcut keymap)
+        command->display-text (fn [command]
+                                (keymap/key-combo->display-text (get command->shortcut command)))
```

Comment:
Bug: `(get command->shortcut command)` may return `nil` if a custom keymap reassigned the shortcut to another command. This will cause `keymap/key-combo->display-text` to throw an exception.

Reply:
thank you, fixed!